Data/Text.hs

```diff
@@ -254,7 +254,11 @@ import GHC.Base (eqInt, neInt, gtInt, geInt, ltInt, leInt)
 import qualified GHC.Exts as Exts
 #endif
 import qualified Language.Haskell.TH.Lib as TH
-import Language.Haskell.TH.Syntax (Lift, lift)
+import Language.Haskell.TH.Syntax ( Lift, lift
+#if MIN_VERSION_template_haskell(2,16,0)
```

Comment:
I'm not a fan (anymore) of splitting up "statements" via CPP; this tends to confuse syntax-highlighters or indenters; please use "complete" statements (i.e. use a 2nd `import` statement guarded by CPP)

or even better yet; let's just use a qualified import `as TH` or similar, and avoid this CPP altogether in this place

Reply:
Done.